src/imitation/scripts/common/rl.py

```diff
@@ -75,6 +75,26 @@ def sac():
     locals()  # quieten flake8
 
 
+def _maybe_add_relabel_buffer(
+    rl_kwargs: Dict[str, Any],
+    relabel_reward_fn: Optional[RewardFn] = None,
+) -> Dict[str, Any]:
+    """Use ReplayBufferRewardWrapper in rl_kwargs if relabel_reward_fn is not None."""
+    _rl_kwargs = dict(rl_kwargs)
```

Comment:
You don't use `rl_kwargs` anywhere in this function after this, so why create a 2nd variable?
```suggestion
    rl_kwargs = dict(rl_kwargs)
```